tests/test_structures.py

```diff
@@ -0,0 +1,73 @@
+import ase
+import numpy as np
+import pytest
+
+
+@pytest.mark.usefixtures("aiida_profile_clean")
+def test_structure_manager_widget(structure_data_object):
+    """Test the `StructureManagerWidget`."""
+    import aiidalab_widgets_base as awb
+
+    structure_manager_widget = awb.StructureManagerWidget(
+        importers=[
+            awb.StructureUploadWidget(title="From computer"),
```

Comment:
Since we are testing the widget here in general, it should be possible to test tab names are correctly set as "From computer" and "Basic Editor".

Reply:
After looking at the code - not really. Both editors and importers are set to local variables. It's not easily possible to access their values 🤷 .

Reply:
Okay, then never bother with that.